fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -130,4 +160,17 @@ public String apply(final Integer type) {
 
     private static final Logger LOGGER = getLogger(DefaultMessageFactory.class);
 
+    private String getPropertiesAsString(final FixityEvent fixityEvent) {
+        final StringBuilder propsStrBuilder = new StringBuilder();
+        for (Map.Entry<String,String> entry : fixityEvent.getInfo().entrySet()) {
+            propsStrBuilder.append(entry.getKey()).append("->").append(entry.getValue()).append(",");
```

Comment:
Could we even have an extra value `BAD_CHECKSUM_AND_BAD_SIZE` to help client code stay clean and simple? Or perhaps use an integer value here with the usual "additive" semantics?